apps/prairielearn/elements/pl-integer-input/pl-integer-input.py

```diff
@@ -68,8 +68,17 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
     else:
         correct_answer = pl.from_json(data["correct_answers"].get(name, None))
 
-    # Test conversion, but leave as string so proper value is shown on answer panel
-    if correct_answer is not None and not isinstance(correct_answer, int):
+    allow_blank = pl.get_boolean_attrib(element, "allow-blank", ALLOW_BLANK_DEFAULT)
+    blank_value = pl.get_string_attrib(element, "blank-value", str(BLANK_VALUE_DEFAULT))
```

Comment:
nit: set `BLANK_VALUE_DEFAULT` to a string instead of doing the conversion here.